server/issue.go

```diff
@@ -1006,3 +1027,36 @@ func (s *Server) postInboxIssueActivity(ctx context.Context, issue *api.Issue, a
 
 	return nil
 }
+
+func (s *Server) getTenantDatabases(ctx context.Context, projectID int, databaseName string) ([][]*api.Database, error) {
+	deployConfig, err := s.DeploymentConfigService.FindDeploymentConfig(ctx, &api.DeploymentConfigFind{
+		ProjectID: &projectID,
+	})
+	if err != nil {
+		return nil, echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to fetch deplopment config for project ID: %v", projectID)).SetInternal(err)
+	}
+	if deployConfig == nil {
```

Comment:
The not found error handling seems to be inconsistent with other places. Other places check common.ErrorCode(err) != common.NotFound